google_compute_engine/metadata_scripts/script_retriever.py

```diff
@@ -111,7 +127,10 @@ def _DownloadScript(self, url, dest_dir):
     # Check for the preferred Google Storage URL format:
     # gs://<bucket>/<object>
     if url.startswith(r'gs://'):
-      return self._DownloadGsUrl(url, dest_dir)
+      # convert it to a regular URL
+      bucket_and_object = url[len(r'gs://'):]
+      url = 'https://storage.googleapis.com/%s' % (bucket_and_object)
```

Comment:
Please remove the extra parens around the variable. In general, use parens sparingly (if they are not needed, don't add them).